CMakeLists.txt

```diff
@@ -223,11 +223,9 @@ endif()
 if(LIBVNCSERVER_HAVE_SYS_UIO_H)
   if(WITH_GCRYPT AND LIBGCRYPT_LIBRARIES)
     message(STATUS "Building crypto with Libgcrypt")
-    set(CRYPTO_LIBRARIES ${LIBGCRYPT_LIBRARIES})
```

Comment:
Bettter be verbose than to rely on side effects. wontmerge

Reply:
it is added a few lines below to ADDITIONAL_LIBRARIES.